src/CommandLine/NServiceBus.Persistence.Sql.CommandLine.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <OutputType>Exe</OutputType>
+    <ToolCommandName>sql-persistence</ToolCommandName>
+    <PackAsTool>True</PackAsTool>
+    <Description>.NET Core global tool to generate scripts for NServiceBus Sql persistence</Description>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="McMaster.Extensions.CommandLineUtils" Version="3.1.0" />
+    <PackageReference Include="Particular.Packaging" Version="0.9.0" PrivateAssets="All" />
```

Comment:
We might be getting this from the `Directory.Build.props` anyway, in which case we shouldn't have it here.

Reply:
removed